Unable to build 3.10-dev (CPython `master` branch renamed to `main`)

As part of the Python 3.10b1 release today, the CPython maintainers renamed the `master` branch to `main` ([briefly breaking the repo](https://github.community/t/renaming-python-master-branch-to-main-1-4k-prs-700-repositories-triggered-server-http-error-500/178090)), so the dev install isn't able to build:

```
$ pyenv install 3.10-dev --verbose
python-build: use openssl@1.1 from homebrew
python-build: use readline from homebrew
/var/folders/y8/cy7tw81s38nbwqdpgz_6pspr0000gn/T/python-build.20210503202054.16193 ~
Cloning https://github.com/python/cpython...
Cloning into 'Python-3.10-dev'...
warning: Could not find remote branch master to clone.
fatal: Remote branch master not found in upstream origin

BUILD FAILED (OS X 11.3 using python-build 20180424)
```

I tried looking around, and I believe the only reference to `master` is here?

https://github.com/pyenv/pyenv/blob/6656066d4fefba64ae247956bb35399f98a399c4/plugins/python-build/share/python-build/3.10-dev#L6

--- a/plugins/python-build/share/python-build/3.10-dev
+++ b/plugins/python-build/share/python-build/3.10-dev
@@ -3,4 +3,4 @@ prefer_openssl11
 export PYTHON_BUILD_CONFIGURE_WITH_OPENSSL=1
 install_package "openssl-1.1.0j" "https://www.openssl.org/source/old/1.1.0/openssl-1.1.0j.tar.gz#31bec6c203ce1a8e93d5994f4ed304c63ccf07676118b6634edded12ad1b3246" mac_openssl --if has_broken_mac_openssl
 install_package "readline-8.0" "https://ftpmirror.gnu.org/readline/readline-8.0.tar.gz#e339f51971478d369f8a053a330a190781acb9864cf4c541060f12078948e461" mac_readline --if has_broken_mac_readline
-install_git "Python-3.10-dev" "https://github.com/python/cpython" master standard verify_py310 copy_python_gdb ensurepip
+install_git "Python-3.10-dev" "https://github.com/python/cpython" main standard verify_py310 copy_python_gdb ensurepip

